README.md

```diff
@@ -38,20 +38,17 @@ All commands:
 
 1. Configure [direv](https://direnv.net/). Then copy `.sample.envrc` to a new `.envrc`. Don't forget to run `direnv allow` after any changes to `.envrc`.
```

Comment:
Do we still need direv? Might remove all this also, just keep everything in the docker compose file.

Reply:
Thats a good question. I do like keeping my env files separate since that would keep the git tree clean when I swap around env variables (I do this quite frequently). Is it conventional to keep everything separate?